### ISSUE TITLE
What to do with non-lunar craters?

### ISSUE THREAD

**Issue by msdemlei** [astrothesaurus/UAT#349]
The UAT has http://astrothesaurus.org/uat/949 ("Lunar craters") but no
analogous concept that would describe craters on non-lunar surfaces.

Given that 949 is firmly grounded in "Lunar Features", generalising it
to cover other craters is probably a non-starter.  So: Do we want
"crater" as a separate concept (which would then have to be "Craters
on anything but Earth's Moon" or so)?

2113 ("Planetary Surfaces") seems a reasonable ancestor concept, though
I have to say none of "Surface Composition", "Surface Variability",
"Surface Ices", and "Surface Processes" strike me as a very natural
direct parent for Crater.  Whatever would be a natural parent could then
perhaps also host the Volcanoes that currently sit (IMHO somewhat
awkwardly) in planetary structure (1256).

Or are we happy with 779 (impact phenomena) for craters?  Perhaps that
could move to 2113 (it's currently a child of 1255 (planetary science))?
Be that as it may, I'd vote for having a "related" relationship between
949 and 779.

**Comment by stlibrary8** [comment]
Some Space Telescope Science Institute folks are also reviewing planetary terms and want an entry for non-lunar craters. Current suggestion is to add child term for Impact phenomena and use "Craters" as preferred term. The thinking is Craters (the product) is more generic than "Cratering" (the process). Could add a "Use for: Cratering" entry with new "Craters" term. Similar to above, should add related term for 949 "Lunar craters" and the new, generic "Craters" term. Agree Planetary Surfaces and related child terms under Surfaces is not ideal place for "[Non-lunar] Craters".

**Comment by katieefrey** [comment]
@msdemlei 

I also agree that we should add a planetary focused craters concept.  Does the solution proposed by the STScI Library above work?

**Comment by msdemlei** [comment]
On Wed, Jun 15, 2022 at 09:22:29AM -0700, Katie Frey wrote:
> I also agree that we should add a planetary focused craters
> concept.  Does the solution proposed by the STScI Library above
> work?  

Sure, though the interesting part will be the definition of the new
concept...

I don't want to burden this issue with working out the relationship
between 2113 and and 779, but in case people in in rough agreement
it's an improvement if impact phenomena becomes a child of planetary
surfaces rather than planetary science, then I'd say we should do it
while we're touching this rough area.
